go.sum

```diff
@@ -274,6 +274,8 @@ github.com/dolthub/ishell v0.0.0-20240701202509-2b217167d718 h1:lT7hE5k+0nkBdj/1
 github.com/dolthub/ishell v0.0.0-20240701202509-2b217167d718/go.mod h1:ehexgi1mPxRTk0Mok/pADALuHbvATulTh6gzr7NzZto=
 github.com/dolthub/jsonpath v0.0.2-0.20240227200619-19675ab05c71 h1:bMGS25NWAGTEtT5tOBsCuCrlYnLRKpbJVJkDbrTRhwQ=
 github.com/dolthub/jsonpath v0.0.2-0.20240227200619-19675ab05c71/go.mod h1:2/2zjLQ/JOOSbbSboojeg+cAwcRV0fDLzIiWch/lhqI=
+github.com/dolthub/pg_extension v0.0.0-20250617034731-28245107b718 h1:QpZ9CNWziqPL5F0daIY028XNA026lUtqf+au4BknGbw=
```

Comment:
I think this is fine as a stop gap to making progress, but I don't think we want to manage this as a separate library / repo long term. Would be nice to get this merged in sooner rather than later. Is there some reason that's not feasible?